pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -75,6 +75,10 @@ type BuildPodImages struct {
 	CompletionWindowsImage string
 }
 
+type volumeCounter struct {
```

Comment:
can we remove this since it's no longer being used

Reply:
whoops